indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -1093,6 +1093,21 @@ public void resetOffsets(@Nonnull DataSourceMetadata resetDataSourceMetadata)
     addNotice(new ResetOffsetsNotice(resetDataSourceMetadata));
   }
 
+  @Override
```

Comment:
Would be nice to add a javadoc here or in the interface to say a few words about the base sequence name and what kind of strings we can expect in it.

You could also link to the javadoc of `TaskGroup.baseSequenceName` if one already exists. If it doesn't, maybe add a getter for this field `TaskGroup.getBaseSequenceName()` and write a small javadoc for it.

Reply:
Done

Reply:
Thanks!